lib/sidekiq/web/helpers.rb

```diff
@@ -323,6 +323,16 @@ def server_utc_time
       Time.now.utc.strftime("%H:%M:%S UTC")
     end
 
+    # Routes of the web UI where Live Poll should not be available
+    NOT_POLLABLE = ["metrics"]
+
+    # Split current_path to be able to match for both "metrics" and "metrics/:name"
+    def pollable?
+      return false if current_path.blank?
+
```

Comment:
return !current_path.starts_with("metrics")

?

Reply:
We also want to disable it for `/sidekiq`, which works with `current_path.blank?`.

But maybe it should be:

```ruby
def pollable?
  return false if current_path.blank? # /sidekiq
  
  NOT_POLLABLE.none? { |path| current_path.starts_with?(path) }
  end
```